serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -17,9 +17,9 @@ This guide will teach you how to build a Serverless Application Container Image
 
 == Prerequisites
 
-Before starting, make sure you have the needed dependencies installed.
+Before starting, make sure you have the needed dependencies installed. +
```

Comment:
```suggestion
Before starting, make sure you have the needed dependencies installed. +
```

Just curious, why do we need this plus sign? What does it do?

Reply:
I made the same question to Tristan :)
Yes.

Reply:
this is forcing a line break: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#paragraphs